### ISSUE TITLE
Development env. setup guidelines

### ISSUE THREAD

**Issue by glung** [pardom/redux-kotlin#11]
Hey, I like very much your impl. but when trying to dig a bit more into it I stumbled upon issues with the IDE. 

The project does not compile w/ IJ : the following imports can't be found. 

``` java
import org.jetbrains.spek.api.Spek
import org.mockito.Mockito.mock
import org.mockito.Mockito.times
import org.mockito.Mockito.verify
```

It would be interesting to either provide setup instructions. 

Note : it does work with gradle in command line.

**Comment by pardom-zz** [comment]
Yes there is an issue with `compileTestKotlin2Js` at the moment where it doesn't recognize your main sources. Additionally, the libraries used don't or can't target `Kotlin2Js`. There's a comment about it in this commit: https://github.com/pardom/redux-kotlin/commit/94025a45c44120cdf229b5f2702140ad6aec4a6c

I do agree, however, that until this is fixed the build script should somehow ignore `compileTestKotlin2Js` or there should be instructions to build with `-x compileTestKotlin2Js`. The tests will run and the build will succeed with the aforementioned task exclusion.

**Comment by glung** [comment]
Oh ok. It makes sense (if you want to target the JVM, Android and JS apps). 
How do you set `-x compileTestKotlin2Js` up in the IDE actually ?

**Comment by pardom-zz** [comment]
Fixed in #12. No special steps are required to build anymore.
